custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -98,25 +98,43 @@ def _state_aggregation_query(self, state_id):
             "(({} - child_health.opened_on::date)::integer >= 0) "
             "AND (child_health.closed = 0 OR (child_health.closed_on::date - {})::integer > 0)"
         ).format(end_month_string, start_month_string)
+        open_in_month_end = (
+            "(({} - child_health.opened_on::date)::integer >= 0) "
+            "AND (child_health.closed = 0 OR (child_health.closed_on::date - {})::integer > 0)"
```

Comment:
can we simplify this logic to something like
```
child_health.opened_on<next_monthstart AND (child_health.closed is null OR child_health.closed>=next_month_start)
```

Reply:
changed